### PR TITLE
Feature/dashboard display apps

### PR DESCRIPTION
**Pull request by zanderlx**
# Issue/Ticket

CLOSES #XX
PROGRESS ON #XX

If you specify CLOSES, the ticket will be automatically closed and moved to 'done' when the PR is merged.

If you specify PROGRESS ON, a reference will be created to that issue (in both directions), but merging the PR will not close the issue.

# Description

Since the endpoint for getting all individual apps was created, there is now functionality for displaying them on the front end. Using a default 55x55 png image and some details about the individual app such as its Title, I have displayed all apps registered with the SSO using Vuetify cards. **(See notes at the bottom)** 

## Type of change

Feature

# How Has This Been Tested?

Manual testing in a localhost environment. 

# Notes

This feature is still a work in progress as **launching to the individual app** doesn't seem to be working. The ability to edit the description of an application is possible but does not persist. (Due to endpoint still being developed) Updating an application's logo is also a work in progress.

### CODE REVIEW

**Comment by alfredovargasjr** [comment]
@zanderlx Pull in the latest changes from Master

**Review by alfredovargasjr — CHANGES_REQUESTED** [review]
Update branch

File: Frontend/src/views/Dashboard.vue [hunk 1]
```diff
@@ -1,61 +1,144 @@
...
-            this.error = "An unexpected server error occurred. Please try again momentarily.";
-            break;
-        }
-      })
+        });
+    }
+  },
+  async mounted() {
+    await axios
+      .get("http://localhost:61348/api/applications")
```
Comment (alfredovargasjr) [inline Frontend/src/views/Dashboard.vue]: set this to the apiURL constant

**Review by alfredovargasjr — CHANGES_REQUESTED** [review]

File: Frontend/src/views/Dashboard.vue (hunk 1 above)
Comment (zanderlx) [reply Frontend/src/views/Dashboard.vue]: I made the change.

File: Frontend/src/views/Dashboard.vue [hunk 2]
```diff
@@ -1,61 +1,145 @@
...
-            this.error = "An unexpected server error occurred. Please try again momentarily.";
-            break;
-        }
-      })
+        });
+    }
+  },
+  async mounted() {
+    await axios
+      .get(`${apiURL}/api/applications`)
```
Comment (alfredovargasjr) [inline Frontend/src/views/Dashboard.vue]: this endpoint is incorrect, .../api/api/applications

**Review by alfredovargasjr — CHANGES_REQUESTED** [review]

File: Frontend/src/views/Dashboard.vue (hunk 2 above)
Comment (zanderlx) [reply Frontend/src/views/Dashboard.vue]: Sorry I didn't see this right away. It should be fixed now

**Comment by alfredovargasjr** [comment]
@zanderlx Resolve the merge conflict

**Comment by zanderlx** [comment]
@alfredovargasjr Fixed the merge conflict

**Review by alfredovargasjr — APPROVED** [review]

**Pull request merged**